src/interfaces/form-interface.ts

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 
-export interface FormIF extends Vue {
+export interface FormIF extends HTMLFormElement {
```

Comment:
This is correct. Now we just need to update this in Filings UI (all the others are correct)... or better yet, just import this shared IF :)

Up to you whether to do it as part of your current ticket or create a new ticket.

Reply:
^^ I'll update Filings UI in a PR I'm already working on.